### PR TITLE
user/asciinema: new package

### PR DESCRIPTION
**Pull request by cassiofb-dev**

### CODE REVIEW

File: user/asciinema/template.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+pkgname = "asciinema"
+pkgver = "2.4.0"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+]
+depends = [
```
Comment (q66) [inline user/asciinema/template.py:10]: one line

File: user/asciinema/template.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "python",
+    "ncurses",
+]
+checkdepends = [
```
Comment (q66) [inline user/asciinema/template.py:14]: ditto

File: user/asciinema/template.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+]
+depends = [
+    "python",
+    "ncurses",
```
Comment (q66) [inline user/asciinema/template.py:12]: what's the purpose of depending on ncurses package, does it need some of the programs?

**Comment by q66** [comment]
> Hi @nekopsykose gm! Did I mess something up? The build is looping on failure and spamming my email 🥲

it's just an microsoft's apt repo mirror moment outside our control

File: user/asciinema/template.py (hunk 3 above)
Comment (nekopsykose) [reply user/asciinema/template.py]: it used to use tput and whatnot until 2.1, seems nobody read the changelog

**Pull request merged**